app/scripts/background.js

```diff
@@ -61,6 +65,21 @@ initialize().catch(log.error)
 // setup metamask mesh testing container
 setupMetamaskMeshMetrics()
 
+/*
+ * The data emitted from the MetaMaskController.store EventEmitter, also used to initialize the MetaMaskController.
+ * @typedef MetaMaskState
```

Comment:
We could potentially describe at the top-level structure of the state here. Something like:

```js
/**
 * The data emitted from the MetaMaskController.store EventEmitter, also used to initialize the MetaMaskController.
 * @typedef MetaMaskState
 * @property {boolean} isInitialized Whether this client is initialized
 * ...
 */
```

It may be overkill though. Just a thought.

Reply:
I agree this is an option, it's just a research task I wasn't fully up to at that moment. That said, it's really nice to have....